<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1185</width>
    <height>821</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="map">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>0</y>
     <width>601</width>
     <height>361</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>340</y>
     <width>581</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>0</y>
     <width>581</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>10</y>
     <width>20</width>
     <height>81</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="line_4">
   <property name="geometry">
    <rect>
     <x>900</x>
     <y>10</y>
     <width>20</width>
     <height>341</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="line_5">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>150</y>
     <width>20</width>
     <height>201</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="line_6">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>240</y>
     <width>20</width>
     <height>111</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="line_7">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>10</y>
     <width>20</width>
     <height>261</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="line_8">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>240</y>
     <width>20</width>
     <height>111</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="line_9">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>230</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_10">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>230</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="QR_1">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>270</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="QR_2">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>270</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="QR_5">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>40</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="QR_4">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>300</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="QR_3">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>70</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="camera">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>311</width>
     <height>361</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>380</y>
     <width>901</width>
     <height>421</height>
    </rect>
   </property>
   <property name="title">
    <string>GroupBox</string>
   </property>
   <widget class="QTableWidget" name="LiveStatus">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>641</width>
      <height>51</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>date</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>temp</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>hum</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>co2</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>dust</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>place</string>
     </property>
    </column>
   </widget>
   <widget class="QTabWidget" name="Place_status">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>591</width>
      <height>311</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="place_1">
     <attribute name="title">
      <string>Place_1</string>
     </attribute>
     <widget class="QTableWidget" name="status_1">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>591</width>
        <height>281</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>date</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>temp</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>hum</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>co2</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>dust</string>
       </property>
      </column>
     </widget>
    </widget>
    <widget class="QWidget" name="place_2">
     <attribute name="title">
      <string>Place_2</string>
     </attribute>
     <widget class="QTableWidget" name="status_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>591</width>
        <height>281</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>date</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>temp</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>hum</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>co2</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>dust</string>
       </property>
      </column>
     </widget>
    </widget>
    <widget class="QWidget" name="place_3">
     <attribute name="title">
      <string>Place_3</string>
     </attribute>
     <widget class="QTableWidget" name="status_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>591</width>
        <height>281</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>date</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>temp</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>hum</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>co2</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>dust</string>
       </property>
      </column>
     </widget>
    </widget>
    <widget class="QWidget" name="place_4">
     <attribute name="title">
      <string>Place_4</string>
     </attribute>
     <widget class="QTableWidget" name="status_4">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>591</width>
        <height>281</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>date</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>temp</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>hum</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>co2</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>dust</string>
       </property>
      </column>
     </widget>
    </widget>
    <widget class="QWidget" name="place_5">
     <attribute name="title">
      <string>Place_5</string>
     </attribute>
     <widget class="QTableWidget" name="status_5">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>591</width>
        <height>281</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>date</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>temp</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>hum</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>co2</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>dust</string>
       </property>
      </column>
     </widget>
    </widget>
   </widget>
   <widget class="QLineEdit" name="feedback">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>100</y>
      <width>261</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btnExport">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>150</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>export</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnReset">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>200</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>reset</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>940</x>
     <y>10</y>
     <width>241</width>
     <height>251</height>
    </rect>
   </property>
   <property name="title">
    <string>GroupBox</string>
   </property>
   <widget class="QPushButton" name="togo_1">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="togo_2">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>40</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="togo_4">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>130</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="togo_3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>130</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="togo_parking">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>210</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Parking</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>940</x>
     <y>270</y>
     <width>231</width>
     <height>221</height>
    </rect>
   </property>
   <property name="title">
    <string>GroupBox</string>
   </property>
   <widget class="QPushButton" name="togo_9">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>100</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>→</string>
    </property>
   </widget>
   <widget class="QPushButton" name="togo_6">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>40</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>↑</string>
    </property>
   </widget>
   <widget class="QPushButton" name="togo_12">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>160</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>↘</string>
    </property>
   </widget>
   <widget class="QPushButton" name="togo_7">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>40</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>↗</string>
    </property>
   </widget>
   <widget class="QPushButton" name="togo_10">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>←</string>
    </property>
   </widget>
   <widget class="QPushButton" name="togo_11">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>160</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>↓</string>
    </property>
   </widget>
   <widget class="QPushButton" name="togo_5">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>↖</string>
    </property>
   </widget>
   <widget class="QPushButton" name="togo_13">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>160</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>↙</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="btnSensor">
   <property name="geometry">
    <rect>
     <x>980</x>
     <y>510</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Sensor on</string>
   </property>
  </widget>
  <widget class="QPushButton" name="test">
   <property name="geometry">
    <rect>
     <x>960</x>
     <y>560</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>test_1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="test_2">
   <property name="geometry">
    <rect>
     <x>1030</x>
     <y>560</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>test_2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="test_3">
   <property name="geometry">
    <rect>
     <x>1100</x>
     <y>560</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>test_3</string>
   </property>
  </widget>
  <widget class="QPushButton" name="test_4">
   <property name="geometry">
    <rect>
     <x>990</x>
     <y>620</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>test_4</string>
   </property>
  </widget>
  <widget class="QPushButton" name="test_5">
   <property name="geometry">
    <rect>
     <x>1070</x>
     <y>620</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>test_5</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnEmer">
   <property name="geometry">
    <rect>
     <x>980</x>
     <y>670</y>
     <width>151</width>
     <height>131</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(249, 240, 107);</string>
   </property>
   <property name="text">
    <string>Emergency Stop</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
